lib/octobox/configurator.rb

```diff
@@ -34,20 +41,21 @@ def restricted_access_enabled
     end
     attr_writer :restricted_access_enabled
 
-    def restricted_access_enabled?
-      restricted_access_enabled
+    def github_organization_id
+      id = @github_organization_id || ENV['GITHUB_ORGANIZATION_ID']
+      return id.to_i if id.present?
     end
+    attr_writer :github_organization_id
+
+    def github_team_id
+      id = @github_team_id || ENV['GITHUB_TEAM_ID']
+      return id.to_i if id.present?
+    end
+    attr_writer :github_team_id
```

Comment:
There are also slight changes in this area; the configuration is now returning an actual integer so the callers don't have to call `to_i` every time.